.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,7 @@
 - [ ] This PR addresses an already opened issue (for bug fixes / features)
     - This PR fixes #xyz
 - [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
+- [ ] This PR does not seem to break the templates.
```

Comment:
I don't think that we actively run them. Could we test it more thoroughly ? If not, I don't know how useful that statement is.

Reply:
My idea was that (for example) if you change the API, you at least check that the template is not broken ? That's why I used "seem".

Testing the templates would be a bit awkward I think...

Reply:
like do you want people to run them locally or just look and feel that they aren't broken ?
maybe we could run them at least each time we have a release ?

Reply:
They can't be run locally as they currently are, sine many things (such as catalog paths) are just placeholders. I'm fine with "just looking", as long as we develop tests eventually.

Reply:
ouais, mais en l'absence de tests pour l'instant,  je suggérais que Pascal et moi (qui avons les paths), on roule les templates avant de passer la version 6.0 mettons.

Reply:
I think I'll add a note in the release process. Manually testing at each PR would be burdensome, maybe that could be done only before publishing. And yes, we'd need to think on how these can be tested...